### PR TITLE
[Move Expando:circluar dependency in local service]Lps 191596

### PR DESCRIPTION
**Pull request by slnn**
https://liferay.atlassian.net/browse/LPS-191596

We need to break these circular dependencies since OSGi Declarative Services doesn’t accommodate circular references.

Fix the original issue in https://github.com/liferay-core-infra/liferay-portal/pull/5167

### CODE REVIEW

File: portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoValueLocalServiceImpl.java [hunk 1]
```diff
@@ -85,10 +83,11 @@ public ExpandoValue addValue(
 			String columnName, long classPK, boolean data)
 		throws PortalException {
 
-		ExpandoTable table = _expandoTableLocalService.getTable(
-			companyId, className, tableName);
+		ExpandoTable table = _expandoTablePersistence.findByC_C_N(
```
Comment (dantewang) [inline portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoValueLocalServiceImpl.java:86]: @slnn Please add an SF commit to rename these variables to follow our convention.

File: portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoValueLocalServiceImpl.java [hunk 2]
```diff
@@ -85,10 +83,11 @@ public ExpandoValue addValue(
...
+			companyId, _classNameLocalService.getClassNameId(className),
+			tableName);
 
-		ExpandoColumn column = _expandoColumnLocalService.getColumn(
+		ExpandoColumn column = _expandoColumnPersistence.fetchByT_N(
```
Comment (dantewang) [inline portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoValueLocalServiceImpl.java:90]: @slnn Why you do null check in some cases and don't in other cases, like this one?

**Review by dantewang — CHANGES_REQUESTED** [review]

File: portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoValueLocalServiceImpl.java (hunk 1 above)
Comment (slnn) [reply portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoValueLocalServiceImpl.java]: Done

**Comment by slnn** [comment]
ci:test:relevant

**Comment by slnn** [comment]
ci:test:sf

File: portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoValueLocalServiceImpl.java (hunk 1 above)
Comment (slnn) [reply portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoValueLocalServiceImpl.java]: Hi @dantewang 

I have another concern about the rename for the variable even though the change has been done, do we need to rename these get methods of these localServices? For example, the `getTable()` method return a instance of ExpandoTable, so if we should also remame the method name to `getExpandoTable()`?
```
ExpandoTable table = _expandoTableLocalService.getTable(
			companyId, className, tableName);
```

**Comment by catarineaguiar** [comment]
@slnn regression found:

```
     [echo] diff --git a/modules/apps/fragment/fragment-api/bnd.bnd b/modules/apps/fragment/fragment-api/bnd.bnd
     [echo] index 8ce7351ed06ae..afcde878c6cb4 100644
     [echo] --- a/modules/apps/fragment/fragment-api/bnd.bnd
     [echo] +++ b/modules/apps/fragment/fragment-api/bnd.bnd
     [echo] @@ -1,6 +1,6 @@
     [echo]  Bundle-Name: Liferay Fragment API
     [echo]  Bundle-SymbolicName: com.liferay.fragment.api
     [echo] -Bundle-Version: 37.0.1
     [echo] +Bundle-Version: 38.0.0
     [echo]  Export-Package:\
     [echo]  	com.liferay.fragment.configuration,\
     [echo]  	com.liferay.fragment.constants,\
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalService.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalService.java
     [echo] index 905baa81fc500..3cc06c68263fd 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalService.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalService.java
     [echo] @@ -94,7 +94,8 @@ public interface ExpandoColumnLocalService
     [echo]  	public PersistedModel createPersistedModel(Serializable primaryKeyObj)
     [echo]  		throws PortalException;
     [echo]  
     [echo] -	public void deleteColumn(ExpandoColumn column) throws PortalException;
     [echo] +	public void deleteColumn(ExpandoColumn expandoColumn)
     [echo] +		throws PortalException;
     [echo]  
     [echo]  	public void deleteColumn(long columnId) throws PortalException;
     [echo]  
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceUtil.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceUtil.java
     [echo] index 399f50fc03873..65c0798912ab9 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceUtil.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceUtil.java
     [echo] @@ -82,10 +82,10 @@ public class ExpandoColumnLocalServiceUtil {
     [echo]  		return getService().createPersistedModel(primaryKeyObj);
     [echo]  	}
     [echo]  
     [echo] -	public static void deleteColumn(ExpandoColumn column)
     [echo] +	public static void deleteColumn(ExpandoColumn expandoColumn)
     [echo]  		throws PortalException {
     [echo]  
     [echo] -		getService().deleteColumn(column);
     [echo] +		getService().deleteColumn(expandoColumn);
     [echo]  	}
     [echo]  
     [echo]  	public static void deleteColumn(long columnId) throws PortalException {
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceWrapper.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceWrapper.java
     [echo] index f194b3606f130..32a8ee21f23a8 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceWrapper.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceWrapper.java
     [echo] @@ -86,10 +86,10 @@ public class ExpandoColumnLocalServiceWrapper
     [echo]  	}
     [echo]  
     [echo]  	@Override
     [echo] -	public void deleteColumn(ExpandoColumn column)
     [echo] +	public void deleteColumn(ExpandoColumn expandoColumn)
     [echo]  		throws com.liferay.portal.kernel.exception.PortalException {
     [echo]  
     [echo] -		_expandoColumnLocalService.deleteColumn(column);
     [echo] +		_expandoColumnLocalService.deleteColumn(expandoColumn);
     [echo]  	}
     [echo]  
     [echo]  	@Override
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalService.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalService.java
     [echo] index e7d3e1cb80fea..34182033746c3 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalService.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalService.java
     [echo] @@ -123,7 +123,7 @@ public interface ExpandoRowLocalService
     [echo]  	public PersistedModel deletePersistedModel(PersistedModel persistedModel)
     [echo]  		throws PortalException;
     [echo]  
     [echo] -	public void deleteRow(ExpandoRow row);
     [echo] +	public void deleteRow(ExpandoRow expandoRow);
     [echo]  
     [echo]  	public void deleteRow(long rowId) throws PortalException;
     [echo]  
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceUtil.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceUtil.java
     [echo] index 1aadd7ff556c6..cee349a790a61 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceUtil.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceUtil.java
     [echo] @@ -117,8 +117,8 @@ public class ExpandoRowLocalServiceUtil {
     [echo]  		return getService().deletePersistedModel(persistedModel);
     [echo]  	}
     [echo]  
     [echo] -	public static void deleteRow(ExpandoRow row) {
     [echo] -		getService().deleteRow(row);
     [echo] +	public static void deleteRow(ExpandoRow expandoRow) {
     [echo] +		getService().deleteRow(expandoRow);
     [echo]  	}
     [echo]  
     [echo]  	public static void deleteRow(long rowId) throws PortalException {
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceWrapper.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceWrapper.java
     [echo] index edd51444a7363..3d72babe3be1f 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceWrapper.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceWrapper.java
     [echo] @@ -120,8 +120,8 @@ public class ExpandoRowLocalServiceWrapper
     [echo]  	}
     [echo]  
     [echo]  	@Override
     [echo] -	public void deleteRow(ExpandoRow row) {
     [echo] -		_expandoRowLocalService.deleteRow(row);
     [echo] +	public void deleteRow(ExpandoRow expandoRow) {
     [echo] +		_expandoRowLocalService.deleteRow(expandoRow);
     [echo]  	}
     [echo]  
     [echo]  	@Override
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalService.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalService.java
     [echo] index 52cf5ef0d2121..d6c2f579ddcf0 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalService.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalService.java
     [echo] @@ -134,7 +134,7 @@ public interface ExpandoTableLocalService
     [echo]  	public PersistedModel deletePersistedModel(PersistedModel persistedModel)
     [echo]  		throws PortalException;
     [echo]  
     [echo] -	public void deleteTable(ExpandoTable table) throws PortalException;
     [echo] +	public void deleteTable(ExpandoTable expandoTable) throws PortalException;
     [echo]  
     [echo]  	public void deleteTable(long tableId) throws PortalException;
     [echo]  
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceUtil.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceUtil.java
     [echo] index d8c2a4c06b4ff..84fc023b8c270 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceUtil.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceUtil.java
     [echo] @@ -139,8 +139,10 @@ public class ExpandoTableLocalServiceUtil {
     [echo]  		return getService().deletePersistedModel(persistedModel);
     [echo]  	}
     [echo]  
     [echo] -	public static void deleteTable(ExpandoTable table) throws PortalException {
     [echo] -		getService().deleteTable(table);
     [echo] +	public static void deleteTable(ExpandoTable expandoTable)
     [echo] +		throws PortalException {
     [echo] +
     [echo] +		getService().deleteTable(expandoTable);
     [echo]  	}
     [echo]  
     [echo]  	public static void deleteTable(long tableId) throws PortalException {
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceWrapper.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceWrapper.java
     [echo] index c2572e3b024b4..267a4bbecee37 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceWrapper.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceWrapper.java
     [echo] @@ -146,10 +146,10 @@ public class ExpandoTableLocalServiceWrapper
     [echo]  	}
     [echo]  
     [echo]  	@Override
     [echo] -	public void deleteTable(ExpandoTable table)
     [echo] +	public void deleteTable(ExpandoTable expandoTable)
     [echo]  		throws com.liferay.portal.kernel.exception.PortalException {
     [echo]  
     [echo] -		_expandoTableLocalService.deleteTable(table);
     [echo] +		_expandoTableLocalService.deleteTable(expandoTable);
     [echo]  	}
     [echo]  
     [echo]  	@Override
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalService.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalService.java
     [echo] index cbe2ae18b4480..0f5bcbb28ce8d 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalService.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalService.java
     [echo] @@ -190,7 +190,7 @@ public interface ExpandoValueLocalService
     [echo]  		throws PortalException;
     [echo]  
     [echo]  	public void addValues(
     [echo] -			long classNameId, long tableId, List<ExpandoColumn> columns,
     [echo] +			long classNameId, long tableId, List<ExpandoColumn> expandoColumns,
     [echo]  			long classPK, Map<String, String> data)
     [echo]  		throws PortalException;
     [echo]  
     [echo] @@ -259,7 +259,7 @@ public interface ExpandoValueLocalService
     [echo]  
     [echo]  	public void deleteTableValues(long tableId);
     [echo]  
     [echo] -	public void deleteValue(ExpandoValue value);
     [echo] +	public void deleteValue(ExpandoValue expandoValue);
     [echo]  
     [echo]  	public void deleteValue(long valueId) throws PortalException;
     [echo]  
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceUtil.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceUtil.java
     [echo] index 2ec45578595b8..644254fbfdd99 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceUtil.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceUtil.java
     [echo] @@ -255,11 +255,12 @@ public class ExpandoValueLocalServiceUtil {
     [echo]  
     [echo]  	public static void addValues(
     [echo]  			long classNameId, long tableId,
     [echo] -			List<com.liferay.expando.kernel.model.ExpandoColumn> columns,
     [echo] +			List<com.liferay.expando.kernel.model.ExpandoColumn> expandoColumns,
     [echo]  			long classPK, Map<String, String> data)
     [echo]  		throws PortalException {
     [echo]  
     [echo] -		getService().addValues(classNameId, tableId, columns, classPK, data);
     [echo] +		getService().addValues(
     [echo] +			classNameId, tableId, expandoColumns, classPK, data);
     [echo]  	}
     [echo]  
     [echo]  	public static void addValues(
     [echo] @@ -353,8 +354,8 @@ public class ExpandoValueLocalServiceUtil {
     [echo]  		getService().deleteTableValues(tableId);
     [echo]  	}
     [echo]  
     [echo] -	public static void deleteValue(ExpandoValue value) {
     [echo] -		getService().deleteValue(value);
     [echo] +	public static void deleteValue(ExpandoValue expandoValue) {
     [echo] +		getService().deleteValue(expandoValue);
     [echo]  	}
     [echo]  
     [echo]  	public static void deleteValue(long valueId) throws PortalException {
     [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceWrapper.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceWrapper.java
     [echo] index 6b1d774de8a4f..e9bf8693e8afe 100644
     [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceWrapper.java
     [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceWrapper.java
     [echo] @@ -276,12 +276,12 @@ public class ExpandoValueLocalServiceWrapper
     [echo]  	public void addValues(
     [echo]  			long classNameId, long tableId,
     [echo]  			java.util.List<com.liferay.expando.kernel.model.ExpandoColumn>
     [echo] -				columns,
     [echo] +				expandoColumns,
     [echo]  			long classPK, java.util.Map<String, String> data)
     [echo]  		throws com.liferay.portal.kernel.exception.PortalException {
     [echo]  
     [echo]  		_expandoValueLocalService.addValues(
     [echo] -			classNameId, tableId, columns, classPK, data);
     [echo] +			classNameId, tableId, expandoColumns, classPK, data);
     [echo]  	}
     [echo]  
     [echo]  	@Override
     [echo] @@ -386,8 +386,8 @@ public class ExpandoValueLocalServiceWrapper
     [echo]  	}
     [echo]  
     [echo]  	@Override
     [echo] -	public void deleteValue(ExpandoValue value) {
     [echo] -		_expandoValueLocalService.deleteValue(value);
     [echo] +	public void deleteValue(ExpandoValue expandoValue) {
     [echo] +		_expandoValueLocalService.deleteValue(expandoValue);
     [echo]  	}
     [echo]  
     [echo]  	@Override
```
full log [here](https://testray.liferay.com/reports/production/logs/2023-09/test-1-40/test-portal-acceptance-pullrequest(master)/5729/service-builder-jdk8/0/0/jenkins-console.txt.gz?authuser=0)

**Comment by slnn** [comment]
> @slnn regression found:
> 
> ```
>      [echo] diff --git a/modules/apps/fragment/fragment-api/bnd.bnd b/modules/apps/fragment/fragment-api/bnd.bnd
>      [echo] index 8ce7351ed06ae..afcde878c6cb4 100644
>      [echo] --- a/modules/apps/fragment/fragment-api/bnd.bnd
>      [echo] +++ b/modules/apps/fragment/fragment-api/bnd.bnd
>      [echo] @@ -1,6 +1,6 @@
>      [echo]  Bundle-Name: Liferay Fragment API
>      [echo]  Bundle-SymbolicName: com.liferay.fragment.api
>      [echo] -Bundle-Version: 37.0.1
>      [echo] +Bundle-Version: 38.0.0
>      [echo]  Export-Package:\
>      [echo]  	com.liferay.fragment.configuration,\
>      [echo]  	com.liferay.fragment.constants,\
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalService.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalService.java
>      [echo] index 905baa81fc500..3cc06c68263fd 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalService.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalService.java
>      [echo] @@ -94,7 +94,8 @@ public interface ExpandoColumnLocalService
>      [echo]  	public PersistedModel createPersistedModel(Serializable primaryKeyObj)
>      [echo]  		throws PortalException;
>      [echo]  
>      [echo] -	public void deleteColumn(ExpandoColumn column) throws PortalException;
>      [echo] +	public void deleteColumn(ExpandoColumn expandoColumn)
>      [echo] +		throws PortalException;
>      [echo]  
>      [echo]  	public void deleteColumn(long columnId) throws PortalException;
>      [echo]  
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceUtil.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceUtil.java
>      [echo] index 399f50fc03873..65c0798912ab9 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceUtil.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceUtil.java
>      [echo] @@ -82,10 +82,10 @@ public class ExpandoColumnLocalServiceUtil {
>      [echo]  		return getService().createPersistedModel(primaryKeyObj);
>      [echo]  	}
>      [echo]  
>      [echo] -	public static void deleteColumn(ExpandoColumn column)
>      [echo] +	public static void deleteColumn(ExpandoColumn expandoColumn)
>      [echo]  		throws PortalException {
>      [echo]  
>      [echo] -		getService().deleteColumn(column);
>      [echo] +		getService().deleteColumn(expandoColumn);
>      [echo]  	}
>      [echo]  
>      [echo]  	public static void deleteColumn(long columnId) throws PortalException {
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceWrapper.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceWrapper.java
>      [echo] index f194b3606f130..32a8ee21f23a8 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceWrapper.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoColumnLocalServiceWrapper.java
>      [echo] @@ -86,10 +86,10 @@ public class ExpandoColumnLocalServiceWrapper
>      [echo]  	}
>      [echo]  
>      [echo]  	@Override
>      [echo] -	public void deleteColumn(ExpandoColumn column)
>      [echo] +	public void deleteColumn(ExpandoColumn expandoColumn)
>      [echo]  		throws com.liferay.portal.kernel.exception.PortalException {
>      [echo]  
>      [echo] -		_expandoColumnLocalService.deleteColumn(column);
>      [echo] +		_expandoColumnLocalService.deleteColumn(expandoColumn);
>      [echo]  	}
>      [echo]  
>      [echo]  	@Override
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalService.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalService.java
>      [echo] index e7d3e1cb80fea..34182033746c3 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalService.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalService.java
>      [echo] @@ -123,7 +123,7 @@ public interface ExpandoRowLocalService
>      [echo]  	public PersistedModel deletePersistedModel(PersistedModel persistedModel)
>      [echo]  		throws PortalException;
>      [echo]  
>      [echo] -	public void deleteRow(ExpandoRow row);
>      [echo] +	public void deleteRow(ExpandoRow expandoRow);
>      [echo]  
>      [echo]  	public void deleteRow(long rowId) throws PortalException;
>      [echo]  
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceUtil.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceUtil.java
>      [echo] index 1aadd7ff556c6..cee349a790a61 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceUtil.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceUtil.java
>      [echo] @@ -117,8 +117,8 @@ public class ExpandoRowLocalServiceUtil {
>      [echo]  		return getService().deletePersistedModel(persistedModel);
>      [echo]  	}
>      [echo]  
>      [echo] -	public static void deleteRow(ExpandoRow row) {
>      [echo] -		getService().deleteRow(row);
>      [echo] +	public static void deleteRow(ExpandoRow expandoRow) {
>      [echo] +		getService().deleteRow(expandoRow);
>      [echo]  	}
>      [echo]  
>      [echo]  	public static void deleteRow(long rowId) throws PortalException {
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceWrapper.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceWrapper.java
>      [echo] index edd51444a7363..3d72babe3be1f 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceWrapper.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoRowLocalServiceWrapper.java
>      [echo] @@ -120,8 +120,8 @@ public class ExpandoRowLocalServiceWrapper
>      [echo]  	}
>      [echo]  
>      [echo]  	@Override
>      [echo] -	public void deleteRow(ExpandoRow row) {
>      [echo] -		_expandoRowLocalService.deleteRow(row);
>      [echo] +	public void deleteRow(ExpandoRow expandoRow) {
>      [echo] +		_expandoRowLocalService.deleteRow(expandoRow);
>      [echo]  	}
>      [echo]  
>      [echo]  	@Override
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalService.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalService.java
>      [echo] index 52cf5ef0d2121..d6c2f579ddcf0 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalService.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalService.java
>      [echo] @@ -134,7 +134,7 @@ public interface ExpandoTableLocalService
>      [echo]  	public PersistedModel deletePersistedModel(PersistedModel persistedModel)
>      [echo]  		throws PortalException;
>      [echo]  
>      [echo] -	public void deleteTable(ExpandoTable table) throws PortalException;
>      [echo] +	public void deleteTable(ExpandoTable expandoTable) throws PortalException;
>      [echo]  
>      [echo]  	public void deleteTable(long tableId) throws PortalException;
>      [echo]  
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceUtil.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceUtil.java
>      [echo] index d8c2a4c06b4ff..84fc023b8c270 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceUtil.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceUtil.java
>      [echo] @@ -139,8 +139,10 @@ public class ExpandoTableLocalServiceUtil {
>      [echo]  		return getService().deletePersistedModel(persistedModel);
>      [echo]  	}
>      [echo]  
>      [echo] -	public static void deleteTable(ExpandoTable table) throws PortalException {
>      [echo] -		getService().deleteTable(table);
>      [echo] +	public static void deleteTable(ExpandoTable expandoTable)
>      [echo] +		throws PortalException {
>      [echo] +
>      [echo] +		getService().deleteTable(expandoTable);
>      [echo]  	}
>      [echo]  
>      [echo]  	public static void deleteTable(long tableId) throws PortalException {
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceWrapper.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceWrapper.java
>      [echo] index c2572e3b024b4..267a4bbecee37 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceWrapper.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoTableLocalServiceWrapper.java
>      [echo] @@ -146,10 +146,10 @@ public class ExpandoTableLocalServiceWrapper
>      [echo]  	}
>      [echo]  
>      [echo]  	@Override
>      [echo] -	public void deleteTable(ExpandoTable table)
>      [echo] +	public void deleteTable(ExpandoTable expandoTable)
>      [echo]  		throws com.liferay.portal.kernel.exception.PortalException {
>      [echo]  
>      [echo] -		_expandoTableLocalService.deleteTable(table);
>      [echo] +		_expandoTableLocalService.deleteTable(expandoTable);
>      [echo]  	}
>      [echo]  
>      [echo]  	@Override
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalService.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalService.java
>      [echo] index cbe2ae18b4480..0f5bcbb28ce8d 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalService.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalService.java
>      [echo] @@ -190,7 +190,7 @@ public interface ExpandoValueLocalService
>      [echo]  		throws PortalException;
>      [echo]  
>      [echo]  	public void addValues(
>      [echo] -			long classNameId, long tableId, List<ExpandoColumn> columns,
>      [echo] +			long classNameId, long tableId, List<ExpandoColumn> expandoColumns,
>      [echo]  			long classPK, Map<String, String> data)
>      [echo]  		throws PortalException;
>      [echo]  
>      [echo] @@ -259,7 +259,7 @@ public interface ExpandoValueLocalService
>      [echo]  
>      [echo]  	public void deleteTableValues(long tableId);
>      [echo]  
>      [echo] -	public void deleteValue(ExpandoValue value);
>      [echo] +	public void deleteValue(ExpandoValue expandoValue);
>      [echo]  
>      [echo]  	public void deleteValue(long valueId) throws PortalException;
>      [echo]  
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceUtil.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceUtil.java
>      [echo] index 2ec45578595b8..644254fbfdd99 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceUtil.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceUtil.java
>      [echo] @@ -255,11 +255,12 @@ public class ExpandoValueLocalServiceUtil {
>      [echo]  
>      [echo]  	public static void addValues(
>      [echo]  			long classNameId, long tableId,
>      [echo] -			List<com.liferay.expando.kernel.model.ExpandoColumn> columns,
>      [echo] +			List<com.liferay.expando.kernel.model.ExpandoColumn> expandoColumns,
>      [echo]  			long classPK, Map<String, String> data)
>      [echo]  		throws PortalException {
>      [echo]  
>      [echo] -		getService().addValues(classNameId, tableId, columns, classPK, data);
>      [echo] +		getService().addValues(
>      [echo] +			classNameId, tableId, expandoColumns, classPK, data);
>      [echo]  	}
>      [echo]  
>      [echo]  	public static void addValues(
>      [echo] @@ -353,8 +354,8 @@ public class ExpandoValueLocalServiceUtil {
>      [echo]  		getService().deleteTableValues(tableId);
>      [echo]  	}
>      [echo]  
>      [echo] -	public static void deleteValue(ExpandoValue value) {
>      [echo] -		getService().deleteValue(value);
>      [echo] +	public static void deleteValue(ExpandoValue expandoValue) {
>      [echo] +		getService().deleteValue(expandoValue);
>      [echo]  	}
>      [echo]  
>      [echo]  	public static void deleteValue(long valueId) throws PortalException {
>      [echo] diff --git a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceWrapper.java b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceWrapper.java
>      [echo] index 6b1d774de8a4f..e9bf8693e8afe 100644
>      [echo] --- a/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceWrapper.java
>      [echo] +++ b/portal-kernel/src/com/liferay/expando/kernel/service/ExpandoValueLocalServiceWrapper.java
>      [echo] @@ -276,12 +276,12 @@ public class ExpandoValueLocalServiceWrapper
>      [echo]  	public void addValues(
>      [echo]  			long classNameId, long tableId,
>      [echo]  			java.util.List<com.liferay.expando.kernel.model.ExpandoColumn>
>      [echo] -				columns,
>      [echo] +				expandoColumns,
>      [echo]  			long classPK, java.util.Map<String, String> data)
>      [echo]  		throws com.liferay.portal.kernel.exception.PortalException {
>      [echo]  
>      [echo]  		_expandoValueLocalService.addValues(
>      [echo] -			classNameId, tableId, columns, classPK, data);
>      [echo] +			classNameId, tableId, expandoColumns, classPK, data);
>      [echo]  	}
>      [echo]  
>      [echo]  	@Override
>      [echo] @@ -386,8 +386,8 @@ public class ExpandoValueLocalServiceWrapper
>      [echo]  	}
>      [echo]  
>      [echo]  	@Override
>      [echo] -	public void deleteValue(ExpandoValue value) {
>      [echo] -		_expandoValueLocalService.deleteValue(value);
>      [echo] +	public void deleteValue(ExpandoValue expandoValue) {
>      [echo] +		_expandoValueLocalService.deleteValue(expandoValue);
>      [echo]  	}
>      [echo]  
>      [echo]  	@Override
> ```
> 
> full log [here](https://testray.liferay.com/reports/production/logs/2023-09/test-1-40/test-portal-acceptance-pullrequest(master)/5729/service-builder-jdk8/0/0/jenkins-console.txt.gz?authuser=0)

Thank @catarineaguiar for catching this! Will fix and resend.

**Pull request closed without merging**